src/threechances.cpp

```diff
@@ -296,14 +342,54 @@ void ThreeChances::collisions() {}
 void ThreeChances::render() {
 	graphics->spriteBegin();
 
-	level.draw();
-	player.draw();
-	duck.draw();
-	ghost.draw();
-	//slug.draw();
-	hud->draw();
-	movesHeader.draw();
-	sword.draw();
+	switch (gameControl->getGeneralState()) {
+		case GENERAL_STATE::menu: {
+			titleFont->Print(
+				GAME_WIDTH / 2 - titleFont->getTotalWidth("three") / 2 - 10,
+				GAME_HEIGHT / 4,
+				"three"
+				);
+			titleFont->Print(
+				GAME_WIDTH / 2 - titleFont->getTotalWidth("chances") / 2 - 10,
+				GAME_HEIGHT /4 + 45,
+				"chances"
+				);
+			secondaryTitleFont->Print(
+				GAME_WIDTH / 2 - secondaryTitleFont->getTotalWidth("press space to start") / 2 - 10,
+				GAME_HEIGHT - 120,
+				"press space to start"
+				);
+
+		} break;
+		case GENERAL_STATE::paused: {
+			titleFont->setScale(0.8f);
+			titleFont->Print(
+				GAME_WIDTH / 2 - titleFont->getTotalWidth("paused") / 2 - 10,
+				GAME_HEIGHT / 4,
+				"paused"
+				);
+			secondaryTitleFont->Print(
+				GAME_WIDTH / 2 - secondaryTitleFont->getTotalWidth("press space to") / 2 - 10,
+				GAME_HEIGHT - 120,
+				"press space to"
+				);
+			secondaryTitleFont->Print(
+				GAME_WIDTH / 2 - secondaryTitleFont->getTotalWidth("continue") / 2 - 10,
+				GAME_HEIGHT - 120 + 45,
```

Comment:
`/2` to align words in the center of screen. `-10` to remove strange padding on the left of screen when setting word. `45` and `120` is the spacing between words at a scale of `x2`.